ui/src/cohortReview/plugins/textSearch.tsx

```diff
@@ -0,0 +1,167 @@
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Stack from "@mui/material/Stack";
+import Typography from "@mui/material/Typography";
+import { useCohortReviewContext } from "cohortReview/cohortReviewContext";
+import {
+  CohortReviewPlugin,
+  registerCohortReviewPlugin,
+} from "cohortReview/pluginRegistry";
+import { Search } from "components/search";
+import { SortDirection, SortOrder } from "data/configuration";
+import { DataValue } from "data/types";
+import { GridBox } from "layout/gridBox";
+import GridLayout from "layout/gridLayout";
+import React, { useMemo } from "react";
+import { CohortReviewPageConfig } from "underlaysSlice";
+
+type Config = {
+  occurrence: string;
+  title: string;
+  text: string;
+  subtitles?: string[];
+  sortOrder?: SortOrder;
+};
+
+@registerCohortReviewPlugin("textSearch")
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class _ implements CohortReviewPlugin {
+  public occurrences: string[];
+  private config: Config;
+
+  constructor(
+    public id: string,
+    public title: string,
+    config: CohortReviewPageConfig
+  ) {
+    this.config = config.plugin as Config;
+    this.occurrences = [this.config.occurrence];
+  }
+
+  render() {
+    return <TextSearch id={this.id} config={this.config} />;
+  }
+}
+
+const fillerText = [
```

Comment:
Yes. For now I'm filling NULL text attributes with one of these generated strings but if there was actual text present it would be displayed. I left a TODO to remove this once we start having data with non-NULL text attributes.